website/content/api-docs/secret/consul.mdx

```diff
@@ -163,6 +179,14 @@ To create a client token with defined roles:
 }
 ```
 
+To create a client token with service identities attached:
+
+```json
+{
+  "service_identities": "myservice-1:dc1,dc2;myservice-2:dc1"
```

Comment:
Same here -- I think this will look different now.

Reply:
You're right, I forgot about the doc changes here. I'm going to add this to my docs-specific task rather than update this PR again. (Same for other doc file). Thanks for catching this!!